CSharp/Library/Microsoft.Bot.Builder/Dialogs/PromptDialog.cs

```diff
@@ -434,16 +434,28 @@ public sealed class PromptConfirm : Prompt<bool, string>
             /// Index of no descriptions.
             /// </summary>
             public const int No = 1;
-
+            
             /// <summary>
             /// The yes, no options for confirmation prompt
             /// </summary>
-            public static string[] Options { set; get; } = { Resources.MatchYes.SplitList().First(), Resources.MatchNo.SplitList().First() };
```

Comment:
removing the setter can be a breaking change. Is it possible to fix the bug without removing the setter?